NuKeeper.Git/LibGit2SharpDriver.cs

```diff
@@ -144,8 +144,11 @@ public void Push(string remoteName, string branchName)
 
             using (var repo = MakeRepo())
             {
-                var localBranch = repo.Branches[branchName];
-                var remote = repo.Network.Remotes[remoteName];
+                
+                var localBranch = repo.Branches
+                    .FirstOrDefault(b => b.CanonicalName.EndsWith(branchName, StringComparison.OrdinalIgnoreCase));
```

Comment:
Same here, we expect but one result and if not it should throw

Reply:
updated